exasol_integration_test_docker_environment/lib/api/spawn_test_environment.py

```diff
@@ -0,0 +1,81 @@
+from typing import Tuple, Optional
+import humanfriendly
+
+from exasol_integration_test_docker_environment.cli.common import set_build_config, set_docker_repository_config, \
+    run_task, generate_root_task
+from exasol_integration_test_docker_environment.cli.options.test_environment_options import LATEST_DB_VERSION
+from exasol_integration_test_docker_environment.lib.api.api_errors import ArgumentConstraintError
+from exasol_integration_test_docker_environment.lib.test_environment.spawn_test_environment_with_docker_db import \
+    SpawnTestEnvironmentWithDockerDB
+
+
+def spawn_test_environment(
+        environment_name: str,
+        database_port_forward: Optional[int] = None,
+        bucketfs_port_forward: Optional[int] = None,
+        db_mem_size: str = "2 GiB",
+        db_disk_size: str = "2 GiB",
+        nameserver: Tuple[str,...] = tuple(),
+        deactivate_database_setup: bool = False,
+        docker_runtime: Optional[str] = None,
+        docker_db_image_version: str = LATEST_DB_VERSION,
+        docker_db_image_name: str = "exasol/docker-db",
+        create_certificates: bool = False,
+        source_docker_repository_name: str = 'exasol/script-language-container',
+        source_docker_tag_prefix: str = '',
+        source_docker_username: Optional[str] = None,
+        source_docker_password: Optional[str] = None,
+        target_docker_repository_name: str = 'exasol/script-language-container',
+        target_docker_tag_prefix: str = '',
+        target_docker_username: Optional[str] = None,
+        target_docker_password: Optional[str] = None,
+        output_directory: str = ".build_output",
```

Comment:
The default value we should also extract into a Constant

Reply:
Done